documentation/docs/getting-started/verifiable_credentials/overview.md

```diff
@@ -9,7 +9,38 @@ keywords:
 - person
 ---
 
+Credentials are statements about an entity, such as properties that the entity posseses or capabilities that they have (like drivers licences, passports, or a person's age). Verifiable Credentials (VCs) are statements about the holder of the credential that can be cryptographically verified by a third party, either online or in person. Additionally, the holder decides what is shared and who it is shared.
```

Comment:
Perhaps avoid "Holder" until introduced. Can be replaced with "Subject" or something more clear.

Reply:
Haha I initially had "subject" for both, but then was re-reading the spec and thought it would be best to use the same terminology. Will change to "(VCs) are statements about the subject (eg. Alice has a drivers licence) that can be cryptographically verified".

This way it doesn't mention "subject", which would technically be misaligned with the spec, or "holder" at all.

Reply:
Going to close this out. Please open a new thread if you'd like to see any other changes.